src/lib/stitching/causality/stitching.ts

```diff
@@ -22,9 +28,57 @@ export const causalityStitchingEnvironment = ({
       extend type AuctionsLotStanding {
         saleArtwork: SaleArtwork
       }
+
+      extend type AuctionsLotState {
+        floorSellingPrice2: Money
+      }
     `,
 
     resolvers: {
+      AuctionsLotState: {
+        floorSellingPrice2: {
+          fragment: gql`
+            ... on AuctionsLotState {
+              internalID
+              floorSellingPrice
+            }
+          `,
+          type: Money,
+          resolve: async (
+            { internalID, floorSellingPrice },
+            _args,
+            context,
+            info
+          ) => {
+            const { currency } = await context.saleArtworkRootLoader(internalID)
+            const { units: minor } = floorSellingPrice
+            const major = await moneyMajorResolver(
+              { cents: minor, currency },
+              {},
+              context,
+              info
+            )
+
+            // return { cents: minor, currency }
+
+            return {
+              currencyCode: currency,
+              minor,
+              major,
+              display: formatMoney(major, symbolFromCurrencyCode(currency)),
+            }
+
+            // return info.mergeInfo.delegateToSchema({
+            //   schema: localSchema,
+            //   operation: "query",
+            //   fieldName: "_do_not_use_money",
```

Comment:
Another option I noticed currently in MP is to add a new root field, purely for internal stitching use, so that we can use `delegateToSchema`. An example of this pattern is the [artist series image](https://github.com/artsy/metaphysics/blob/0b4ebf1dd09e6a515f8294dd0e480c2465e65a4a/src/lib/stitching/gravity/stitching.ts#L271). But I'm not sure if we should keep adding root fields for this purpose.